tests/test_csms.py

```
@@ -0,0 +1,506 @@
+  products = csms.merge(list(certs), list(vendors), {}, claims)
+  return csms.sort_entries(products)
+
+
+def directory_row(entries, index=0):
+  """One rendered directory row, keyed by column name."""
+  return dict(zip(csms.TABLE_HEADERS, csms.render_row(entries[index])))
+
+
+def feature_row(entries, index=0):
+  """One rendered annex row, keyed by column name."""
+  return dict(zip(csms.FEATURE_TABLE_HEADERS, csms.render_feature_row(entries[index])))
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)